cmd/flags.go

```diff
@@ -132,41 +129,18 @@ func FlagVersion(version *bool) {
 }
 
 func FlagLogLevel(level *string) {
-	flag.StringVar(level, "log-level", DefaultLogLevel, "set the logging level")
+	flag.StringVar(level, "log-level", envOrDefault(EnvLogLevel, DefaultLogLevel), "logging level")
```

Comment:
does "set the" or similar get added somewhere else in the help/description. "logging level" on it's own doesn't add any value of "log-level".

Reply:
We changed it to be consistent with the other flag help texts which set things but don't begin with "set the". I'm not sure "set the logging level" or "logging level" add anything over "log-level", but I am happy to change it back or just remove the help text entirely.

Reply:
/shrug idk, given how few people use lifecycle directly it probably doesn't matter rn